docs/UserGuide.md

```diff
@@ -5,36 +5,45 @@ title: User Guide
 
 AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
 
-* Table of Contents
-{:toc}
+## Table of contents
+
+- [Quickstart](#Quickstart)
+- [Features](#Features)
+    - [Viewing help](#Viewing-help-help)
+    - [Adding clients](#Adding-a-client-client-add)
+    - [Viewing clients](#Viewing-clients-client-view)
+    - [Finding clients](#Finding-clients-client-find)
+    - [Updating clients](#Updating-a-client-client-update)
+    - [Deleting clients](#Deleting-a-client-client-delete)
+    - [Saving data](#Saving-data)
+    - [Adding notes](#Adding-notes-for-a-country)
+    - [Deleting notes](#Deleting-a-client-client-delete)
+    - [Updating client notes](#Updating-client-notes-client-note-update)
+    - [Filtering by country](#Filtering-clients-by-country-country-filter)
+    - [Adding country notes](#Adding-notes-for-a-country-country-note)
+    - [Clearing all entries](#Clearing-all-entries-clear)
+    - [Exiting the program](#Exiting-the-program--exit)
 
 --------------------------------------------------------------------------------------------------------------------
 
-## Quick start
+## Quickstart
 
-1. Ensure you have Java `11` or above installed in your Computer.
+1. Ensure you have Java 11
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
+2. Download the application file (.jar file) from [here](#https://github.com/AY2021S1-CS2103T-F11-4/tp/releases)
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+3. Double-click the file to start the app
```

Comment:
I think we should retain the step of copying our jar file over to the folder the user wants to use as the home folder, as we'll need to do saving in a data folder as well